buildpipeline/pipelines.json

```diff
@@ -352,6 +391,11 @@
             "GitHubRepositoryName": "coreclr",
             "AzureContainerPackageGlob": "pkg\\*.nupkg",
             "AzureContainerSymbolPackageGlob": "symbolpkg\\*.nupkg"
+          },
+          "ReportingParameters": {
+            "SubType":  "Publish",
+            "Type": "build/publish/",
+            "PB_BuildType": "Release"
```

Comment:
This should get the publish step out of the 'unknown' category in Mission Control